backend/index.ts

```diff
@@ -1,5 +1,40 @@
-import type { _PublicKeyAuthenticatorProtocol } from '../types/interface'
-export interface CreateAuthenticatorOptions {}
-export function createPublicKeyAuthenticator(opts: CreateAuthenticatorOptions): _PublicKeyAuthenticatorProtocol {
-    return {}
+import type { PublicKeyAuthenticatorProtocol } from '../types/interface'
+
+export interface NormalizedCreateOptions {
+    keys: CryptoKeyPair
```

Comment:
It should not appears in the normalized options because that's options was the normalized version that's provided by the user